pkg/operator/ceohelpers/common.go

```diff
@@ -31,3 +37,27 @@ func IsSingleNodeTopology(infraLister configv1listers.InfrastructureLister) (boo
 
 	return topology == configv1.SingleReplicaTopologyMode, nil
 }
+
+func MemberToNodeInternalIP(member *etcdserverpb.Member) (string, error) {
+	memberURLAsString, err := memberToURL(member)
+	if err != nil {
+		return "", err
+	}
+	memberURL, err := url.Parse(memberURLAsString)
+	if err != nil {
+		return "", err
+	}
+
+	host, _, err := net.SplitHostPort(memberURL.Host)
+	if err != nil {
+		return "", err
+	}
+	return host, nil
+}
+
+func memberToURL(member *etcdserverpb.Member) (string, error) {
+	if len(member.PeerURLs) == 0 {
+		return "", fmt.Errorf("unable to extract member's URL address, it has an empty PeerURLs field, member: %v", spew.Sdump(member))
```

Comment:
spewdump member in an error looks too long to me.  How about an ID or an IP or something short.

Reply:
done